internal/pkg/cli/errors.go

```diff
@@ -9,6 +9,29 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/term/color"
 )
 
+type errCannotDowngradePipelineVersion struct {
+	name            string
+	version         string
+	templateVersion string
+}
+
+func (e *errCannotDowngradePipelineVersion) init() *errCannotDowngradeVersion {
+	return &errCannotDowngradeVersion{
+		componentName:         e.name,
+		componentType:         "pipeline",
+		currentVersion:        e.version,
+		latestTemplateVersion: e.templateVersion,
```

Comment:
nit nit nit: feel free to ignore this if you disagree! I felt like ⬇️  naming might be clearer...? `currentVersion` could mean "the current deployed template version" or "the current latest version", and `latestTemplateVersion` could mean the latest version of all time, or the latest version of "this release". Anyway, after not reviewing PRs for a while, I got a little confused with the naming 😂 hence the suggesetion. But again, this is probably just me, so feel free to ignore.
```go
type errCannotDowngradeVersion struct {
   laterVersion string // previously `currentVersion`.
   thisVersion string // previously `latestTemplateVersion`.
}
```